src/content/community/translations.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Translations
+---
+
+<Intro>
+
+React docs are translated by the global community into many languages all over the world.
+
+</Intro>
+
+## Main site {/*main-site*/}
```

Comment:
Instead on "Main Site" what do you think about these options:
- Have it list the "Current Site" so for each translation site it lists the one you're on
- Or, move english into "Translated languages" and remove this heading

The issue with "main site" to me is that it seems to diminish the translated sites as if they're less official or important, but they are official and just as valid as the main site

Reply:
English is not a "_translated_ language" but _is_ the main site from which other translations are based on. So I think semantically the current page is correct.

Reply:
Maybe "Canonical site" with text like "All translations are provided from the canonical source docs:"

Reply:
Or "Source site"

Reply:
I'll apply this and merge